src/transformState.js

```diff
@@ -4,8 +4,41 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+const removeKey = function(actionObject, object) {
```

Comment:
also, it is a bad practice to add the dataType to the variable's name. It basically does not give us any information, just takes space. 

I would stick to just `actions`

Reply:
also, `object` is not the best way either. Think of a better way - your names should be maximally clear and descriptive

Reply:
they are just 'data' now